File: deleteatendarray.cpp
# include <iostream>
using namespace std;
int main()
{
  int array[20], n;
  cout<< "Enter the size of array: ";
  cin>> n ; 
  cout<< "Enter the array elements: ";
  for(int i = 0 ; i<n ; i++){
    cin>> array[i];
  }
  n--;
  cout<<"Number after deletion: ";
  for(int i = 0 ; i<n ; i++){
    cout<< array[i] <<endl;
  }
  return 0 ;
}

File: deleteatendarray.java
import java.util.Scanner;

public class deleteatendarray {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int[] array = new int[20];
        int n;

        System.out.print("Enter the size of the array: ");
        n = scanner.nextInt();

        System.out.print("Enter the array elements: ");
        for (int i = 0; i < n; i++) {
            array[i] = scanner.nextInt();
        }

        n--;

        System.out.println("Array after deletion:");
        for (int i = 0; i < n; i++) {
            System.out.println(array[i]);
        }
    }
}
